app/components/events.hbs

```diff
@@ -0,0 +1,24 @@
+<section data-test-events-section class='events-section'>
+  <h2 data-test-events-section-header class='events-section__heading'>You can
+    check out all our events below!!</h2>
+  {{#each-in this.EVENTS_CATEGORIES as |category events|}}
+    <div class='events__category'>

```

Comment:
Let's take a example here where you have named `events__category` but our block name is `events-section` so it should be `events-section__category`

Reply:
Similarly see in other places you've did something like this

Reply:
Thank you for the example!
I have updated the class names accordingly.